### PR TITLE
Add crowd SSO functionality

### PR DESCRIPTION
**Pull request by patricktoelle**
Add crowd's single sign on feature to confluence. When using delegated authentication via Atlassian Crowd, you are able to activate this feature according to Atlassian's documentation at https://confluence.atlassian.com/display/CROWD/Integrating+Crowd+with+Atlassian+Confluence#IntegratingCrowdwithAtlassianConfluence-2.2EnableSSOintegrationwithCrowd

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 fixtures:
+  repositories:
+    stdlib: "git://github.com/puppetlabs/puppetlabs-stdlib"
+    deploy: "https://github.com/mkrakowitzer/puppet-deploy.git"
+    staging: "https://github.com/nanliu/puppet-staging.git"
```
Comment (igalic) [inline .fixtures.yml]: is there a specific reason for this change?

Comment (patricktoelle) [inline .fixtures.yml]: this was done by mistake, I'll checkout that file from master.

**Comment by jyaworski** [comment]
Can you squash?

**Comment by igalic** [comment]
and rebase…
****
relatedly, this project needs ~~standard~~ labels

**Comment by patricktoelle** [comment]
Just squashed my changes into one commit. Should be fine now.

**Comment by jyaworski** [comment]
@patricktoelle I'm seeing merge conflicts.

**Comment by patricktoelle** [comment]
Hi, just resolved the merge conflict.

**Comment by patricktoelle** [comment]
Oh, rubocop Tests are failing. I'll have a look on that later.

**Comment by patricktoelle** [comment]
Tests are working now, please merge :)

**Pull request merged**